config/bootscripts/boot-odroid-xu4-next.cmd

```diff
@@ -1,93 +1,24 @@
-# This is an example file to generate boot.scr - a boot script for U-Boot
```

Comment:
I would have prefered to use ${fdtfile} ${initrdname} ${kernelname} but it doesn't work, only /boot/zImage /boot/uInitrd /boot/dtb/exynos5422-odroidxu4.dtb work properly